docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -226,120 +231,233 @@ An example of these causes from the :ref:`large-scale-food fortification concept
+
+.. math::
+  :nowrap:
+
+  \begin{align*}
+  &i &&= \text{identifier for an individual simulant}\\
+  &c &&= \text{identifier for a cause}\\
+  &\text{cat}(i) &&= \text{low birth weight short gestation category of individual $i$}\\
+  &\text{state}_c(i) &&= \text{current cause state of individual $i$ in cause model diagram for $c$}\\
+  &\text{CSMR}_c &&= \text{cause-specific mortality rate for cause $c$}\\
+  &\text{EMR}_{\text{state}_c(i)} &&= \text{excess mortality rate for the cause state state$_c(i)$}\\
+  &\textit{RR}_{\text{cat}(i)} &&= \text{relative risk for all-cause mortality in LBWSG category cat$(i)$}\\
+  &\text{PAF} &&= \text{PAF of LBWSG for affected causes at most-detailed cause level}
+  \end{align*}
+
+See the :ref:`note below <PAF information>` about how to compute the above PAF.
+Note that since :math:`\text{state}_c(i)` implicitly depends on the time
+:math:`t`, the individual mortality hazard will also depend on time. Using the
+above definitions, we will define the following individual mortality rates
+below:
+
+.. math::
+  :nowrap:
+
+  \begin{align*}
+  &\text{csmr}_c(i) &&= \text{unstratified cause-specific mortality hazard from cause $c$ for $i$}\\
+  &\text{csmr}_c^*(i) &&= \text{LBWSG-stratified cause-specific mortality hazard from $c$ for $i$}\\
+  &\text{mr}(i) &&= \text{overall mortality hazard for individual $i$}
+  \end{align*}
+
+For each cause :math:`c`, define the unstratified cause-specific mortality
+hazard for individual :math:`i` to be
+
+.. math::
+
+  \text{csmr}_c(i) :=
+  \begin{cases}
+  \text{CSMR}_c
+    & \text{if $c \in$ unmodelled}, \\
+  \text{EMR}_{\text{state}_c(i)}
+    & \text{if $c\in $ modelled}.
+  \end{cases}
+
+Note that "unstratified" here means unstratified with respect to LBWSG category.
+The above individual csmr's can in fact be interpreted as the cause-level CSMR's
+stratified on all the individual cause states observed in the simulation. Now
+define the LBWSG-stratified cause-specific mortality hazard of :math:`c` for
+individual :math:`i` to be
+
+.. math::
+
+  \text{csmr}_c^*(i) :=
+  \begin{cases}
+  \text{csmr}_c(i)
+    & \text{if $c \in$ unaffected}, \\
+  \text{csmr}_c(i)\cdot (1-\text{PAF})\cdot \textit{RR}_{\text{cat}(i)}
+    & \text{if $c \in$ affected}.
+  \end{cases}
+
+As described above, we are applying the PAF and relative risks only to the
+causes GBD considers affected by LBWSG. Then the individual's total mortality
+hazard, stratified by all modeled cause states and LBWSG risk categories, is
+
+.. math::
+  :label: mortality_hazard
+
+  \text{mr}(i)
+  & := \sum_{c\,\in\, \text{causes}} \text{csmr}_c^*(i) \\
+  &= \sum_{c\,\in\, \text{pink}}
+    \text{CSMR}_c
+    + \sum_{c\,\in\, \text{salmon}}
+    \text{EMR}_{\text{state}_c(i)} \\
+    &\qquad\qquad + \left(\sum_{c\,\in\, \text{blue}}
+    \text{CSMR}_c
+    + \sum_{c\,\in\, \text{green}}
+    \text{EMR}_{\text{state}_c(i)}\right)
+    \cdot (1-\text{PAF})\cdot \textit{RR}_{\text{cat}(i)},
```

Comment:
since you were able to get a multi-line equation here maybe we can use "modelled, unaffected", etc. rather than color names now? either way works though

Reply:
Yeah, I think I could get it to look decent using "affected, modelled", etc. in the subscripts. But I thought it would be good to explicitly make the connection to the colors defined in the table above, unless we want to omit the colors entirely.

Reply:
sounds good! I'm happy with it as is

Reply:
You better keep the colors! I spent ages on it :) hahaha